yarn.lock

```diff
@@ -7821,12 +7821,12 @@
     "@udecode/plate-table" "10.6.3"
     validator "13.7.0"
 
-"@udecode/plate-serializer-html@23.7.0":
-  version "23.7.0"
-  resolved "https://registry.yarnpkg.com/@udecode/plate-serializer-html/-/plate-serializer-html-23.7.0.tgz#94a3b1cc566f57cfd73092f8ee7f7e11085b3f23"
-  integrity sha512-XlwXvSv4+SkDr3dJAj02OKj5bHY0xDoGs6nLbRtsnSsp5Wf9wJ0wCgjI/AAcsVhO6jE67MQvwWtDFKOANvvLSA==
+"@udecode/plate-serializer-html@30.1.2":
```

Comment:
@aodhagan-cf 👀

Reply:
we should just have 30 now but I'm not sure why there would even be changes here, I'd say pull from master and install again